tests/core/test_core.py

```diff
@@ -472,22 +472,28 @@ def test_stream_muting_confirmation_popup(
         [
             ([], [["search", "FOO"]]),
             ([["search", "BOO"]], [["search", "FOO"]]),
-            ([["stream", "PTEST"]], [["stream", "PTEST"], ["search", "FOO"]]),
+            ([["stream", "Stream 1"]], [["stream", "Stream 1"], ["search", "FOO"]]),
             (
-                [["pm_with", "foo@zulip.com"], ["search", "BOO"]],
-                [["pm_with", "foo@zulip.com"], ["search", "FOO"]],
+                [["pm_with", "person1@example.com"], ["search", "BOO"]],
+                [["pm_with", "person1@example.com"], ["search", "FOO"]],
             ),
             (
-                [["stream", "PTEST"], ["topic", "RDS"]],
-                [["stream", "PTEST"], ["topic", "RDS"], ["search", "FOO"]],
+                [["stream", "Stream 1"], ["topic", "RDS"]],
+                [["stream", "Stream 1"], ["topic", "RDS"], ["search", "FOO"]],
```

Comment:
Are these changes just tidying, or required? If tidying, better to keep in a separate commit.

Reply:
They are required changes. The `final_narrow` is created only when the narrow exists in the org, else the test case fails.